tests/system/providers/google/cloud/gcs/example_gcs_to_gcs.py

```diff
@@ -263,9 +242,7 @@ def delete_work_dir(create_workdir_result: str) -> None:
         sync_to_subdirectory,
         sync_from_subdirectory,
         copy_single_file,
-        copy_files_with_wildcard,
         copy_files_without_wildcard,
```

Comment:
The name of this task will lose its meaning after removing `copy_files_with_wildcard`.  You could rename it to just `copy_files` or another name that you think is more fitting.